packages/cli/src/commands/validatorgroup/rpc-urls.ts

```diff
@@ -0,0 +1,93 @@
+import { concurrentMap, StrongAddress } from '@celo/base'
+import { ClaimTypes, IdentityMetadataWrapper } from '@celo/metadata-claims'
+import { Flags, ux } from '@oclif/core'
+import { BaseCommand } from '../../base'
+
+export default class RpcUrls extends BaseCommand {
+  static description =
+    'Displays a list of community RPC nodes for the currently elected validator groups'
+
+  static aliases: string[] = [
+    'network:rpc-urls',
```

Comment:
i tried to do something like this before and could not get it to work. i wonder if i just forgot to rebuild the oclif manifest